navit/maptool/osm.c

```diff
@@ -1103,6 +1119,41 @@ void osm_add_tag(char *k, char *v) {
             flags[0] |= AF_SPEED_LIMIT;
         level=5;
     }
+    if (!g_strcmp0(k, "maxheight")) {
```

Comment:
Think we need to deal with imperial units as well. See https://wiki.openstreetmap.org/wiki/Key:maxheight
Same for the other dimensions.